package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "itoolkit",
-  "version": "0.1.6",
-  "description": "Wrapper over XMLSERVICE for access to all things IBM i",
+  "version": "1.0.0",
```

Comment:
Shouldn't updating the version be a separate PR?

Reply:
Ok will remove